Sources/Sentry/SentryCoreDataTracker.m

```diff
@@ -44,23 +44,18 @@ - (NSArray *)managedObjectContext:(NSManagedObjectContext *)context
     }];
 
     if (fetchSpan) {
-        [SentryLog
-            logWithMessage:[NSString stringWithFormat:
-                                         @"SentryCoreDataTracker automatically "
-                                         @"started a new span with description: %@, operation: %@",
-                                     fetchSpan.description, SENTRY_COREDATA_FETCH_OPERATION]
-                  andLevel:kSentryLevelDebug];
+        SENTRY_LOG_DEBUG(@"SentryCoreDataTracker automatically started a new span with "
+                         @"description: %@, operation: %@",
+            fetchSpan.description, fetchSpan.operation);
```

Comment:
Converted all the logs to the new macros. Also here's the second place I changed interpolating the operation constant for SentrySpan.operation.